pkg/start/start.go

```diff
@@ -291,6 +291,10 @@ func watchHostAgentEvents(ctx context.Context, inst *store.Instance, haStdoutPat
 				return true
 			}
 
+			if xerr := runAnsibleProvision(ctx, inst); xerr != nil {
```

Comment:
Can we have an integration test in https://github.com/lima-vm/lima/blob/master/hack/test-templates/test-misc.yaml ?